### PR TITLE
Initial Tweaks

### PR DESCRIPTION
**Pull request by kingthorin**
As 0discussed on slack.

- Add license.
- Add 2.0 PDF locally.
- Migrate project about info from wiki.owasp.org
- Add "Builder" and repo to right nav (info.md).
- Updated leaders.md

image

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -5,49 +5,20 @@ title: OWASP Code Review Guide
 tags: Code Review Guide
 level: 3
 type: documentation
-auto-migrated: 1
 
 ---
 
-The current PDF version can be found [here](/www-pdf-archive/OWASP_Code_Review_Guide_v2.pdf)
+[![OWASP Flagship](https://img.shields.io/badge/owasp-lab-yellow.svg)](https://owasp.org/projects/#all-projects)
+[![CC BY-SA 3.0](https://licensebuttons.net/l/by-sa/3.0/80x15.png)](https://creativecommons.org/licenses/by-sa/3.0/)
```
Comment (ThunderSon) [inline index.md:12]: Let's bump this version 4 and update the license, what say you?

File: index.md [hunk 2]
```diff
@@ -5,49 +5,20 @@ title: OWASP Code Review Guide
...
 
-<!-- Standard Chapter Page Template
-This is an example of a Project or Chapter page.
-Please change these items to indicate the actual information you wish to present. In addition to this information, the 'front-matter' above the text should be modified to reflect your actual information.  An explanation of each of the front-matter items is below:
+The current (July 2017) PDF version can be found [here](assets/OWASP_Code_Review_Guide_v2.pdf)
 
-{front matter for this file}
+OWASP Code Review Guide is a technical book written for those responsible for code reviews (management, developers, security professionals). The primarily focus of this book has been divided into two main sections. Section one is why and how of code reviews and sections two is devoted to what vulnerabilities need to be to look for during a manual code review. While security scanners are improving every day the need for manual security code reviews still needs to have a prominent place in organizations SDLC (Secure development life cycle) that desires good secure code in production.
```
Comment (ThunderSon) [inline index.md:16]:
```suggestion
OWASP Code Review Guide is a technical book written for those responsible for code reviews (management, developers, security professionals). The primarily focus of this book has been divided into two main sections. Section one is the "why and how of code reviews" and section two focuses on the "types of vulnerabilities and how to identify throughout the review". While security scanners are improving every day the need for manual security code reviews still needs to have a prominent place in organizations' SDLC (Secure Development Life Cycle) that desires good secure code in production.
```

File: index.md [hunk 3]
```diff
@@ -5,49 +5,20 @@ title: OWASP Code Review Guide
...
 
-```
-- layout: This is the layout used by project and chapter pages.  You should leave this value as col-sidebar
-- title: This is the title of your project or chapter page, usually the name.  For example, OWASP Zed Attack Proxy or OWASP Baltimore
-- tags: This is a space-delimited list of tags you associate with your project or chapter.  If you are using tabs, at least one of these tags should be unique in order to be used in the tabs files (an example tab is included in this repo) 
-- region: This is the region you are in according to our data
-```
+Second sections deals with vulnerabilities. It is based on the poplar OWASP 2013 top 10. Here you will find most of the code examples for both on what not to do and on what to do. A word of caution on code examples; Perl is famous for its saying that there are 10,000 ways to do one thing. The same is true for C#, PHP and Java or any other computer language. Now add in "Object-Oriented Programming" and if we are using design patterns or even what designs patterns are being used and sample code becomes very “iff” in what to write. We tried to keep the sample code so code reviews can see red flags and not “do it my way or else”.
```
Comment (ThunderSon) [inline index.md:18]:
```suggestion
Chapters in the second section are mostly based on the popular OWASP 2013 top 10. Here you will find most of the code examples for both on "what not to do" and on "what to do". A word of caution on code examples; Perl is famous for its saying that there are 10,000 ways to do one thing. The same is true for C#, PHP and Java or any other computer language. Now add in "Object-Oriented Programming" and if we are using design patterns or even what designs patterns are being used and sample code becomes very “iff” in what to write. We tried to keep the sample code so code reviewers can see red flags and not “do it my way or else”.
```

File: index.md [hunk 4]
```diff
@@ -5,49 +5,20 @@ title: OWASP Code Review Guide
...
 
-{copy for this file (index.md)}
-Replace the text above the commented area with your information in the format below:
-```
-## Welcome
-Include some information here about your chapter
+The last section is the appendix. Here we have content like code reviewer check list, etc. of items that really don’t flow in book form but needed to be included to make the code review guide compete.
```
Comment (ThunderSon) [inline index.md:20]:
```suggestion
The last section is the appendix. Here we have content like code reviewer check list, etc. of items that really don't flow in book form but needed to be included to make the code review guide complete. 
```

File: index.md [hunk 5]
```diff
@@ -5,49 +5,20 @@ title: OWASP Code Review Guide
...
-
-This separate file is where you should place links to your Google Group and Meetup page. It will be automatically rendered in the column sidebar.
-
-{leaders.md}
-
-Another separate file that should simply include each leaders name with mailto link as a list. It will also be automatically rendered in the column sidebar.
-
--->
\ No newline at end of file
+OWASP Code Review Guide is free to use. It is licensed under the [Creative Commons Attribution-ShareAlike 3.0 license](http://creativecommons.org/licenses/by-sa/3.0/), so you can copy, distribute and transmit the work, and you can adapt it, and use it commercially, but all provided that you attribute the work and if you alter, transform, or build upon this work, you may distribute the resulting work only under the same or similar license to this one.
```
Comment (ThunderSon) [inline index.md:24]: CC4?

File: index.md [hunk 6]
```diff
@@ -5,49 +5,20 @@ title: OWASP Code Review Guide
 tags: Code Review Guide
 level: 3
 type: documentation
-auto-migrated: 1
 
 ---
 
-The current PDF version can be found [here](/www-pdf-archive/OWASP_Code_Review_Guide_v2.pdf)
+[![OWASP Flagship](https://img.shields.io/badge/owasp-lab-yellow.svg)](https://owasp.org/projects/#all-projects)
```
Comment (ThunderSon) [inline index.md:11]: Should be OWASP Lab no?

**Review by ThunderSon — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (kingthorin) [reply index.md:12]: That's fine with me

File: index.md (hunk 5 above)
Comment (kingthorin) [reply index.md:24]: Will

File: index.md (hunk 6 above)
Comment (kingthorin) [reply index.md:11]: Yup

Comment (kingthorin) [reply index.md:11]: Oh you mean the link, Doesn't seem like linking to the lab heading works.

Comment (ThunderSon) [reply index.md:11]:
```suggestion
[![OWASP Lab](https://img.shields.io/badge/owasp-lab-yellow.svg)](https://owasp.org/projects/#all-projects)

```

**Comment by ThunderSon** [comment]
When you just tweak that Flagship to Lab, it'll be good to merge.

File: index.md (hunk 6 above)
Comment (kingthorin) [reply index.md:11]: Oh Tks!

**Review by ThunderSon — APPROVED** [review]

**Pull request merged**